lib/twitter-ads/campaign.rb

```diff
@@ -9,7 +9,7 @@ class Campaign
 
     attr_reader :account
 
-    property :id, read_only: true
+    property :id
```

Comment:
Noted! Looks like we did have this properly implemented and never saw it before, thanks for the hint!!